product-service-description/compliance-as-a-service/features/supported-databases/entity-framework.md

```diff
@@ -16,3 +16,5 @@ Dieser Storage-Provider eignet sich besonders zum Aufbau ausfallsicherer Systeme
 | Name | Beschreibung | Optional |
 | ---- | ------------ |--------- |
 | _connectionstring_ | EF-kompatibler Connection-String zum Datenbanksystem | nein | 
+| _allowCors_ | Kann auf `true` gesetzt werden, um CORS-Zugriffe via REST auf die Middleware zuzulassen. | ja (Default: `false`) | 
```

Comment:
Isn´t this a Launcher parameter?

Reply:
Nope, it's a queue parameter - since we also define timeouts and messagesize via that, I thought it makes sense to use the same approach. What do you think?